R/utilities-mass-balance.R

```diff
@@ -35,7 +33,7 @@ plotMeanMassBalance <- function(structureSet,
   })]
 
   # User defined coumpound selection
-  selectedCompoundNames <- selectedCompoundNames %||% relevantCompoundNames
+  selectedCompoundNames <- settings$selectedCompoundNames %||% relevantCompoundNames

```

Comment:
will this not crash if `settings `is NULL?

Reply:
No, if `settings` is NULL or empty list, `settings$...` will be also NULL

Reply:
ok. didn't know it works like this in R :)